app/controllers/v1/notice_of_disagreements_controller.rb

```diff
@@ -38,5 +39,9 @@ def create
     def error_class(method:, exception_class:)
       "#{self.class.name}##{method} exception #{exception_class} (NOD_V1)"
     end
+
+    def version_number
+      'v2'
+    end
```

Comment:
we originally had a ternary in `AppealSubmission#submit_nod` to give us the version based on the class, but for that to work in v0, we would need to require/import `DecisionReviewsV1::Service` (there was an "uninitialized constant" error in the v0 request spec) so this allows us to sidestep that problem. this is a temporary measure and will likely get ripped out once we are finished with the re-release